packages/related-content/props.js

```diff
@@ -0,0 +1,85 @@
+const baseProps = {
+  headline: {
+    type: 'string',
+    required: true,
+    description: 'Headline for the section',
+  },
+  description: {
+    type: 'string',
+    description: 'Description text for the section',
+  },
+  cards: {
+    type: 'array',
+    required: true,
+    description: 'See <Card /> below in Sub-component-specific Props',
+  },
+  cta: {
+    type: 'string',
+    description: 'Optional CTA',
+  },
+  appearance: {
+    type: 'string',
+    options: ['light', 'dark'],
+    description: 'Render on light or dark backgrounds',
+  },
+}
+
+const cardProps = {
+  children: {
+    type: 'function',
+    description: 'Children to render into the Card.',
+    required: true,
+  },
+  thumbnail: {
+    type: 'object',
+    description: "An image showcasing the card's destination.",
+    properties: {
+      src: {
+        description: 'URL for the image',
+        type: 'string',
+        control: { type: 'input' },
+        required: true,
+      },
+      alt: {
+        description: 'Alternative text for the image',
+        type: 'string',
+        control: { type: 'input' },
+        required: false,
+      },
+    },
+  },
+  meta: {
+    type: 'array',
+    description: 'Content shown above the card heading.',
+    required: false,
+    testValue: ['August 15, 2022', 'Category'],
+  },
+  appearance: {
+    description: 'Styles the card with either a light or dark theme.',
+    type: 'string',
+    required: false,
+    options: ['light', 'dark'],
+    testValue: 'light',
+  },
```

Comment:
If we decide to keep the card docs here, we do not use the appearance for the card object, it takes the appearance from RelatedContent component prop.

Reply:
🔪 [here](https://github.com/hashicorp/react-components/pull/731/commits/8f7ee6a9f5540c9e4788bc7a1e38cfb3fcc89ae6)